src/pages/Devices/index.js

```diff
@@ -1,8 +1,14 @@
 import React from 'react'
 import DeviceList from '../../components/DeviceList'
-
-const Devices = props => {
-    return <DeviceList {...props} />
+import { DevicesContext } from 'lib/DevicesProvider'
+const Devices = () => {
+    return (
+        <div>
```

Comment:
Is the div really needed here or can we directly return the DevicesContext.Consumer?

Reply:
Removed, it was a leftover